Velaptor/PublicExtensionMethods.cs

```diff
@@ -261,5 +288,280 @@ public static GlyphMetrics ApplySize(this GlyphMetrics value, float size)
+        ///             <item>Value of <c>C:/my-directory/.txt</c> will return <c>true</c></item>
+        ///         </list>
+        ///     </item>
+        /// </list>
+        /// </example>
+        /// <remarks>
+        ///     This does not check if the path actually exists.  This is for valid path syntax checks only.
+        /// </remarks>
+        public static bool IsInvalidFilePath(this string path) => !IsValidFilePath(path);
+
+        /// <summary>
+        /// Returns a value indicating if the given <paramref name="path"/> a valid drive.
+        /// </summary>
+        /// <param name="path">The path to check.</param>
+        /// <returns>True if <paramref name="path"/> contains a valid drive.</returns>
+        /// <example>
+        /// <list type="number">
+        ///     <item>Value of <c>C:/my-dir</c> will return <c>true</c>.</item>
+        ///     <item>Value of <c>C:my-dir</c> will return <c>true</c>.</item>
+        ///     <item>Value of <c>C/my-dir/my-file.txt</c> will return <c>false</c>.</item>
+        ///     <item>Value of <c>//my-dir</c> will return <c>false</c>.</item>
+        ///     <item>Value of <c>./my-dir</c> will return <c>false</c>.</item>
+        ///     <item>Value of <c>../my-dir</c> will return <c>false</c>.</item>
+        /// </list>
+        /// </example>
+        /// <remarks>
+        ///     This does not check if the drive actually exists.  This is for valid path syntax checks only.
+        /// </remarks>
+        public static bool ContainsValidDrive(this string path)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                return false;
+            }
+
+            if (path.Length < 2)
+            {
+                return false;
+            }
+
+            if (path.Contains(':') is false)
+            {
+                return false;
+            }
+
+            if (path.StartsWith(':'))
+            {
+                return false;
+            }
+
+            return !path[0].IsNotLetter() && path[1].IsNotLetter();
+        }
+
+        /// <summary>
+        /// Returns a value indicating if the given <paramref name="dirPath"/> is valid.
+        /// </summary>
+        /// <param name="dirPath">The directory path to check.</param>
+        /// <returns>True if the <paramref name="dirPath"/> is valid.</returns>
+        /// <example>
+        /// <list type="number">
+        ///     <item>Value of <c>C:/my-dir</c> will return <c>true</c>.</item>
+        ///     <item>Value of <c>C:/my-dir/</c> will return <c>true</c>.</item>
+        ///     <item>Value of <c>C:/my-dir/my-file.txt</c> will return <c>false</c>.</item>
+        /// </list>
+        /// </example>
+        /// <remarks>
+        ///     This does not check if the directory path actually exists.  This is for valid path syntax checks only.
+        /// </remarks>
+        public static bool IsFullyQualifiedDirPath(this string dirPath)
```

Comment:
Refactor name to below and adjust usages and XML code comments as necessary

```suggestion
        public static bool HasValidFullDirPathSyntax(this string dirPath)
```